components/String/Dumper.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace PHPDoc\Internal\String;
+
+use InvalidArgumentException;
+
+class Dumper
+{
+    public static function dump($item): string
+    {
+        return static::toString($item);
+    }
+
+    private static function toString($item, int $tab = 0): string
+    {
+        return match (gettype($item)) {
```

Comment:
```suggestion
        return match (get_debug_type($item)) {
```
Utilise les noms canoniques (bon faut adapter les branches)

Reply:
Peut-être:
```php
return match (get_debug_type($item)) {
            'string' => '"'.$item.'"',
            'int', 'float' => $item,
            'null' => 'null',
            'boolean' => $item ? 'true' : 'false',
            'array' => static::arrayToString($item, $tab + 2),
            $item::class => "Class instance: ".$item::class,
            default => throw new InvalidArgumentException(sprintf('Use of invalid type "%s"', $item)),
        };
```

Reply:
Ou quelque chose du genre `"Class: ".$item::class`, `"Object: ".$item::class`

Reply:
Object est peut être mieux